source/hic_hal/stm32/stm32h743ii/extended_features/source/DAP_vendor_ex.c

```diff
@@ -271,6 +272,18 @@ uint32_t DAP_ProcessVendorCommandEx(const uint8_t *request, uint8_t *response) {
         num += 1;
         break;
     }
+    case ID_DAP_VendorEx39_READ_FLEX_HOTPLUG_ADC:
+    {
+        uint16_t flex_hotplug_adc_value = get_flex_hotplug_adc_value();
+        *response = DAP_OK;
+        *response++ = 2;
+        *response++ = ((flex_hotplug_adc_value >> 8) & 0xff);
+        *response++ = (flex_hotplug_adc_value & 0xff);
```

Comment:
Out of curiosity, do you know that the response buffer has this many bytes available to write into without overrun?

Reply:
The response buffer is defined by the macro DAP_PACKET_SIZE, which is 64 bytes.